nvflare/apis/fl_constant.py

```diff
@@ -453,6 +456,14 @@ class FLMetaKey:
     SITE_NAME = "site_name"
     PROCESS_RC_FILE = "_process_rc.txt"
     SUBMIT_MODEL_NAME = "submit_model_name"
+    AUTH_TOKEN = "auth_token"
+    AUTH_TOKEN_SIGNATURE = "auth_token_signature"
+
+
+class CellMessageAuthHeaderKey:
+    CLIENT_NAME = "client_name"
```

Comment:
Should this have underscore just like others?

Reply:
This is from existing definition, and want to keep it intact to avoid potential breaks of the code (in case it was hard-coded somewhere).